Documentation/rust/coding-guidelines.rst

```diff
@@ -162,7 +162,7 @@ in the kernel:
     a section called ``# Examples``.
 
   - Rust items (functions, types, constants...) must be linked appropriately
-    (``rustdoc`` will create a link automatically).
+    by wrapping them with square brackets (e.g. ``[`Foo::bar()`]``).
```

Comment:
It is nice adding the explanation, but I think we should also keep the original sentence.

This should be an independent commit, too.

Reply:
Thank you! I have removed these changes from the commit. Should I submit them as a second PR?

Reply:
You're welcome! I think they would be 3 independent PRs, since they are mostly independent (i.e. the cleanup, the docs improvement and the proposal to document private items; the latter likely being an RFC). In any case, please note that each change that you want to submit will need to end up in the mailing list -- we don't really use PRs anymore, except for early reviews like this one. Thanks!